src/nnvm/graph_dump.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2019 by Contributors
+ * \file graph_dump.h
+ * \brief Utilities to introspect and print the execution graph
+ * \author Pedro Larroy
+ */
+
+#ifndef NNVM_GRAPH_DUMP_H_
+#define NNVM_GRAPH_DUMP_H_
+#pragma once
```

Comment:
Those are 2 ways to do the same thing (and pragma once is a nonstandard one).

Reply:
good point, I think I tried pragma once first as is supported by all major compilers but I'm not sure if nvcc supports it.